src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1094,6 +1107,42 @@ namespace winrt::TerminalApp::implementation
         return profileMenuItem;
     }
 
+    // Method Description:
+    // - This method creates a flyout menu item for a given action
+    //   It makes sure to set the correct icon, keybinding, and click-action.
+    WUX::Controls::MenuFlyoutItem TerminalPage::_CreateNewTabFlyoutAction(const winrt::hstring& actionId)
```

Comment:
~nit: we could probably just pass the action in directly here~

eh, we need to look up the keybinding here too. I take it back, this is fine.